docs/getting-started.md

```diff
@@ -0,0 +1,117 @@
+
+```
+terminal WS: /\s+/;
+terminal ID: /[_a-zA-Z][\w]*/;
+```
+
+Here we define our two needed terminals for this grammar: The whitespace `WS` and identifier `ID` terminals. Terminals parse a part of our document by matching it against their regular expression. The `WS` terminal parses any whitespace characters with the regex `/\s+/`. This allows us consume whitespaces in our document. As the terminal is referenced by the grammar as a hidden terminal, the parser will parse any whitespace and discard the results. That way, we don't have to care about how many whitespaces a user uses in their document. Secondly, we define our `ID` terminal. As per its regex, it parses any string that starts with an underscore or letter and continues with any amount of characters that match the `\w` regex token. It will match `Alice`, `_alice`, or `_al1c3` but not `4lice` or `#alice`. Langium is using the JS regex dialect for terminal definitions.
+
+```
+Model: (persons+=Person | greetings+=Greeting)*;
+```
+
+The `Model` parser rule is the entry point to our grammar. Parsing always starts with the entry rule. Here we define a repeating group of alternatives: `persons+=Person | greetings+=Greeting`. This will always try to parse either a `Person` or a `Greeting` and add it to the respective list of `persons` or `greetings` in the `Model` object. Since the alternative is wrapped in a repeating group `*`, the parser will continue until all input has been consumed. 
+
+```
+Person: 'person' name=ID;
+```
+
+The `Person` rule starts off with the `'person'` keyword. Keywords are like terminals, in the sense that they parse a part of the document. The set of keywords and terminals create the tokens that your language is able to parse. You can imagine that the `'person'` keyword here is like an indicator to tell the parser that an object of type `Person` should be parsed. After the keyword, we assign the `Person` a name by parsing an `ID`.
+
+```
+Greeting: 'Hello' person=[Person] '!';
+```
+
+Like the previous rule, the `Greeting` starts with a keyword. With the `person` assignment we introduce the _cross reference_, indicated by the brackets `[]`. A cross reference will allow your grammar to reference other elements that are contained in your file or workspace. By default, Langium will try to resolve this cross reference by parsing an `ID` and looking for a `Person` whose `name` property matches the parsed `ID`.
+
+That finishes the short introduction to Langium! You can continue with the [tutorials](./tutorials.md) or take a closer look at the [grammar](./features/grammar.md) and [abstract syntax tree](./features/ast.md) documentation.
```

Comment:
Maybe it would be nice to tell here that one can change or add anything to the grammar and regenerate the project with `npm run langium:generate`.